advanced/Scripts/piholeDebug.sh

```diff
@@ -330,8 +330,7 @@ function dumpPiHoleLog {
 
 # Anything to be done after capturing of pihole.log terminates
 function finalWork {
-	echo "::: Finshed debugging!" 
-	echo "::: Debug log can be found at : /var/log/pihole_debug.log"
+	echo "::: Debug log can be found at : $(cat /var/log/pihole_debug.log | curl --silent -F 'sprunge=<-' http://sprunge.us)"
```

Comment:
Agreed, quite often we have the situation where the resolver isn't happy, and if the resolver isn't happy then uploads aren't going to work. Right now I get a `curl: (6) Could not resolve host: sprunge.us` response if the DNS is down.